test.md

```diff
@@ -0,0 +1 @@
+"This file should be ignored by git".
```

Comment:
- [ ] Just as the text in this file says this file should be ignored by git and should not be present in your repo, According to the requirements _**A test.md file with the text "This file should be ignored by git".
Make sure that the last file is actually ignored by git and is not present in your GitHub repository.**_